src/internals/StreamHttpReply.cpp

```diff
@@ -18,25 +18,44 @@ ArduinoHttpServer::AbstractStreamHttpReply::AbstractStreamHttpReply(Stream& stre
 
 }
 
+void ArduinoHttpServer::AbstractStreamHttpReply::sendHeader(
+    size_t size, const String& title) {
+   // Read away remaining bytes.
+   while (getStream().read() >= 0) {
+   }
+
+   getStream().print(AHS_F("HTTP/1.1 "));
+   getStream().print(getCode() + " ");
+   getStream().print(title + "\r\n");
+   getStream().print(AHS_F("Connection: keep-alive\r\n"));
```

Comment:
This might be a breaking change. What client do you use? In my case it could be fine (NodeJs / request as client) but cannot recall anymore I experienced stability / robustness or other errors not closing the connection.

Reply:
Yeah sorry, I'm not sure how that snuck in there. I was trying keep-alive to troubleshoot why I was getting TCP resets in the browser. My client is the standard Arduino `EthernetClient`.